i3/4-i3.md

```diff
@@ -0,0 +1,55 @@
+# Introduction to i3-4.24
+i3 is an easy-to-use tiling window manager for X11.
+
+## Package Information
+- Download (HTTP): https://github.com/i3/i3/archive/refs/tags/4.24.tar.gz
+- Download md5sum: 7982a7dd61313e330797be9a633abe4d
+- Download Size: 4.3M
+- Build Size: 15M
+- Build Time: 0.1 SBU
+
+## i3 Dependencies
+### Required
+  [libev](./3-libev),
+  [libxkbcommon](https://www.linuxfromscratch.org/blfs/view/svn/general/libxkbcommon.html),
+  [pango](https://www.linuxfromscratch.org/blfs/view/svn/x/pango.html),
+  [startup-notification](https://www.linuxfromscratch.org/blfs/view/svn/x/startup-notification.html),
+  [XCB Utils](https://glfs-book.github.io/glfs/shareddeps/xcb-utils.html),
```

Comment:
If xcb-util-errors is not needed, let's link to https://linuxfromscratch.org/blfs/view/svn/x/xcb-utilities.html instead.